identity/policies_test.go

```diff
@@ -0,0 +1,453 @@
+package identity_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/joyent/triton-go/identity"
+	"github.com/joyent/triton-go/testutils"
+)
+
+var (
+	listPoliciesErrorType = errors.New("Error executing ListPolicies request:")
```

Comment:
Even though this is new and matching the style of the rest of the package, can we use this as the beachhead to start using lowercase error messages?